DoctrineMongoDBBundle.php

```diff
@@ -78,4 +79,22 @@ public function shutdown()
         spl_autoload_unregister($this->autoloader);
         $this->autoloader = null;
     }
+
+    private function registerCommandLoggers() : void
+    {
+        $commandLoggerRegistry = $this->container->get('doctrine_mongodb.odm.command_logger_registry');
+        $commandLoggerRegistry->register();
+    }
+
+    private function unregisterCommandLoggers() : void
+    {
+        $commandLoggerRegistry = $this->container->get('doctrine_mongodb.odm.command_logger_registry');
+        $commandLoggerRegistry->unregister();
+    }
+
+    public function shutdown()
```

Comment:
Should this have a return type hint, or is that omitted because we're overriding a method from Symfony's base class?

Reply:
I've omitted it because Symfony doesn't have one and I haven't had time to test this against Symfony 5 so far. All Doctrine Bundles will get the Symfony 4.4/5.0 treatment after the Symfony Code Freeze next week (DoctrineBundle has been the guinea pig for this). This will add some return type annotations where applicable. I'll have to see how to treat this since I don't want to hold 4.0 back until November, but I also don't want to release a new major version shortly after releasing 4.0 already.